wynnset/tapestry-wp
# 602 - Add Tapestry settings to initial load

Closes #602 

**Testing**

* Create a new Tapestry
* Check the network tab, find the request to GET a Tapestry
* Look into the response, the settings should include the following fields: `backgroundUrl`, `autoLayout`, `nodeDraggable`, `showAccess`, `defaultPermissions`.
